.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+env:
+  - DB=MYSQL CORE_RELEASE=3.1
+
+matrix:
+  include:
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.5
+
+
+before_script:
+  - composer self-update
```

Comment:
this should be `composer self-update || true` - previously we've had problems where composer was unavailable and so this line failed and tests wouldn't run.

composer comes pre-installed and so this update is just a nice-to-have and shouldn't block tests if it fails